test/jdk/java/lang/instrument/ATransformerManagementTestCase.java

```diff
@@ -117,9 +117,13 @@ public ATransformerManagementTestCase(String name)
         {
             fTransformers.add(transformer);
         }
+        // workaroud for JDK-8264667: create log message before addTransformer()
+        String msg = "Added transformer " + transformer
+            + " with canRetransform=" + canRetransform;
         manager.addTransformer(transformer, canRetransform);
-        verbosePrint("Added transformer " + transformer
-            + " with canRetransform=" + canRetransform);
+//        verbosePrint("Added transformer " + transformer
+//            + " with canRetransform=" + canRetransform);
```

Comment:
I'd suggest to remove the commented out lines.

Reply:
fixed